tests/release/README.md

```diff
@@ -0,0 +1,29 @@
+# Release tests
+
+Contains E2E tests related to [Release Service](https://github.com/redhat-appstudio/infra-deployments/tree/main/components/release).
+
+Steps to run tests within `release` directory:
+
+1. Follow the instructions from the [Readme](../../docs/Installation.md) scripts to install AppStudio in e2e mode
```

Comment:
these are the instructions for openshift-ci, I think the link should point to https://github.com/redhat-appstudio/e2e-tests/blob/main/README.md#install-appstudio-in-e2e-mode

Reply:
after all we changed this README to include the use cases covered in suite 
to avoid duplication , thanks @psturc